docs/getting-started/connect-widget.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Connect Widget"
+description: "Overview of the configurations for the widget."
+---
+
+# Link the the Metriport Connect widget in your app
+
+With Metriport, you get a pre-built [Connect widget](https://connect.metriport.com/?token=demo)
+that you can easily plug into your app, so that your users can connect all of their data sources to your application!
+Here's a preview:
+
+<iframe
+  width="90%"
+  height="700"
+  src="https://connect.metriport.com/?token=demo"
+></iframe>
+<br />
+
+# Url Parameters
+
+Set url params to validate the current session of the widget and make
+certain customizations to fit your product needs.
+
+### token
+
+This is the token generated by metriport. Check the [quick start](/getting-started/connect-quickstart) for more details.
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}`;
+```
+
+### colorMode
+
+You can update the color mode to be either light or dark.<br/>
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}&colorMode=dark`;
+```
+
+<iframe
+  width="90%"
+  height="700"
+  src="https://connect.metriport.com/?token=demo&colorMode=dark"
+></iframe>
+
+### customColor
+
+You can set your own color as the highlight color by adding a default color such as orange or add a hex code.
+For a comprehensive list of available colors [visit here](https://www.w3schools.com/cssref/css_colors.php).<br/>
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}&customColor=orange`;
+```
+
+<Tip>
+  If you wish to add a hex code ensure that it is URL encoded ie.
+  customColor=%23FF5733
+</Tip>
+
+<iframe
+  width="90%"
+  height="700"
+  src="https://connect.metriport.com/?token=demo&customColor=orange"
+></iframe>
+
+### providers
+
+You can choose the providers you wish to display to your users. It must be a comma separated lowercased list of providers.
+For a list of the available providers [visit here](/more-info/our-integrations).<br/>
```

Comment:
just checking, intentionally without a new line between `You can choose...` and `For a list...`?

And need the `<br/>`?